pom.xml

```diff
@@ -72,6 +89,11 @@
                     <linkXRef>false</linkXRef>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <version>3.3.1</version>
+            </plugin>
```

Comment:
does it work without `web.xml`?

Reply:
![image](https://user-images.githubusercontent.com/73486745/129874737-edf3cf4b-0a57-4d2b-b58e-404121a317a9.png)
yes